ibu-imager/seedcleanup/seedcleanup.go

```diff
@@ -0,0 +1,224 @@
+			s.log.Errorf("error disabling %s unit: %v", systemdUnit, err)
+		}
+	}
+	s.log.Info("Preparation systemd units disabled successfully.")
+
+	filesToRemove := []string{
+		"/etc/systemd/system/installation-configuration.env",
+		"/var/usrlocal/bin/installation-configuration.sh",
+		"/var/usrlocal/bin/prepare-installation-configuration.sh",
+		"/etc/systemd/system/installation-configuration.service",
+		"/etc/systemd/system/prepare-installation-configuration.service",
+	}
+
+	for _, file := range filesToRemove {
+		if err := os.Remove(file); err != nil {
+			s.log.Errorf("error removing %s: %v", file, err)
+		}
+	}
+	s.log.Info("Preparation systemd scripts removed successfully.")
+
+	return nil
+}
+
+func (s *SeedCleaner) restoreCertificates() error {
+	etcdImage, err := s.ops.GetImageFromPodDefinition(etcdStaticPodFile, etcdStaticPodContainer)
+	if err != nil {
+		return fmt.Errorf("error getting available etcd image: %w", err)
+	}
+
+	s.log.Info("Run unauthenticated etcd server for recert tool")
+	_, err = s.ops.RunInHostNamespace(
+		"podman", []string{"run", "--name", "recert_etcd",
+			"--detach", "--rm", "--network=host", "--privileged", "--replace",
+			"--authfile", s.authFile, "--entrypoint", "etcd",
+			"-v", "/var/lib/etcd:/store",
+			etcdImage,
+			"--name", "editor",
+			"--data-dir", "/store"}...)
+	if err != nil {
+		return fmt.Errorf("failed to run recert_etcd container: %w", err)
+	}
+
+	defer func() {
+		if _, err = s.ops.RunInHostNamespace(
+			"podman", []string{"kill", "recert_etcd"}...); err != nil {
+			s.log.Errorf("Failed to kill recert_etcd container: %v", err)
+		}
+		s.log.Info("Unauthenticated etcd server and recert containers removed successfully.")
+	}()
+
+	s.log.Info("Waiting for unauthenticated etcd start serving for recert tool")
+	err = s.ops.WaitForEtcd("http://localhost:2379/health")
+	if err != nil {
+		return fmt.Errorf("failed to wait for unauthenticated etcd server: %w", err)
+	}
+	s.log.Info("Unauthenticated etcd server for recert is up and running")
+
+	args := []string{"-f", backupCertsDir + "/ingresskey-*"}
+	ingressKey, err := s.ops.RunBashInHostNamespace("readlink", args...)
+	if err != nil {
+		return err
+	}
+
+	args = []string{ingressKey, "|", "cut", "-d", "-", "-f", "2-"}
+	ingressCN, err := s.ops.RunBashInHostNamespace("basename", args...)
+	if err != nil {
+		return err
+	}
+
+	s.log.Info("Run recert --extend-expiration tool")
+	_, err = s.ops.RunInHostNamespace(
+		"podman", []string{"run", "--name", "recert",
+			"--rm", "--network=host", "--privileged", "--replace", "--authfile", s.authFile,
+			"-v", backupCertsDir + ":/certs",
+			"-v", "/etc/kubernetes:/kubernetes",
+			"-v", "/var/lib/kubelet:/kubelet",
+			"-v", "/etc/machine-config-daemon:/machine-config-daemon",
+			s.recertContainerImage,
+			"--etcd-endpoint", "localhost:2379",
+			"--static-dir", "/kubernetes",
+			"--static-dir", "/kubelet",
+			"--static-dir", "/machine-config-daemon",
```

Comment:
```suggestion
			"--static-dir", "/kubernetes",
			"--static-dir", "/kubelet",
			"--static-dir", "/machine-config-daemon",
```

We should not be repeating this list in multiple places, it should be in a constant somewhere. Also you're missing a `--static-file /host-etc/mcs-machine-config-content.json`

Reply:
added #92 to aim at solving this